lib/src/web_socket.dart

```diff
@@ -56,9 +56,7 @@ class WebSocket {
 
   bool get _isConnected {
     final connectionState = _connectionController.state;
-    return connectionState is Connected ||
-        connectionState is Reconnected ||
-        connectionState is Disconnecting;
+    return connectionState is Connected || connectionState is Reconnected;
```

Comment:
It's technically still connected when Disconnecting so I believe it's correct as is.

Reply:
Agree, no issue occurred, thanks for explanation. close the PR.